plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/commit/CommitPresenter.java

```diff
@@ -76,7 +71,7 @@
     private final ProcessesPanelPresenter consolesPanelPresenter;
 
     private Project      project;
-    private Set<String>  allFiles;
+    private List<String> allFiles;
```

Comment:
Could you please explain why we're changing a set for a list? We need to store same files several times?